scale-tests/setup_streaming.py

```diff
@@ -57,9 +55,12 @@
 log = logging.getLogger(__name__)
 
 
+SUPPORTED_MULTIPLE_CLUSTER_SERVICES = ['kafka', 'confluent-kafka', 'beta-kafka']
+
+
 def install_package(package_name: str, service_prefix: str, index: int, service_task_count: int, config_path: str,
                     additional_options: dict=None) -> dict:
-    service_name = "{}-{:0>2}".format(service_prefix, index)
+    service_name = "{}-{}-{:0>2}".format(service_prefix, package_name, index)
```

Comment:
Although I understand the need for the `package_name` here, the default behaviour will now result in `kafka-kafka-00` (as an example). I would propose one or more of the following:

- Stripping the `beta-` from the package name (not critical)
- only applying `service_prefix` if it is not the same as `package_name` or non-`beta` part of package name.
- Possibly introducing folders?

Reply:
That's a good point. The idea behind prefixes has been adding the possibility to use folders from the beginning (so that the infrastructure and dispatchers from a given test are "isolated" under a shared folder), but there were a few warts that I don't remember now. What I'll do is:
- If the user hasn't specified a prefix, use only the package name
- Try adding the possibility to use folders and see if it works.

I think keeping `beta-` in the package name is actually good... so it's explicit. I don't care strongly about it though, so we can remove it if you do.

Reply:
How about:
```python
    if package_name.startswith("beta-"):
        basename = package_name[len("beta-"):]
    else:
        basename = package_name

    service_name = "{}{}-{:0>2}".format(service_prefix, basename, index)
```
This means that we can easily control the use of folders through the `service_prefix` by setting it to `/some/path/`, for example. I don't have a strong feeling about the `beta-` prefix, so we may as well included it (it does break from the usual behaviour in our packages though).